src/AddTodoBtn.js

```diff
@@ -0,0 +1,17 @@
+
+
+function AddTodoBtn({onAddTodo}) {
+
+    this.addTodoForm = document.querySelector('.add-todo-form');
+    this.addTodoInput = document.querySelector('.add-todo-input');
+
+    this.addTodoForm.addEventListener('submit', (e) => {
+        e.preventDefault();
+        onAddTodo(this.addTodoInput.value);
```

Comment:
보통 리액트에서는 이벤트 핸들러 함수에 대해 `'handle' + 동작 + 대상`의 방식으로 네이밍 하는게 일반적이더라구요!
```suggestion
        handleAddTodoItem(this.addTodoInput.value);
```